### ISSUE TITLE
CallExpression has missing edge when chaining a method call immediately after object initialization

### ISSUE THREAD

**Issue by dentoir** [Fraunhofer-AISEC/cpg#318]
Hi there,

Given the code snippet below, the CallExpression node found on line 3 yields an empty result set for the `getInvokes()` method. The single CallExpression node with code contents `new Dummy().getInteger();` is actually of type `StaticCallExpression`, but I expected there to be a `MemberCallExpression` which would have an outward edge towards the `getInteger()` method in class `Dummy`. 

```
public class Test {
    public void method() {
        int i = new Dummy().getInteger();
        System.err.println(i);
    }
    private class Dummy {
        public int getInteger() {
            return 2;
        }
    }
}
```

This may be related to issue #197 ?

**Comment by oxisto** [comment]
Did you try the 3.3.1 release? This sounds like a duplicate of #285, which was fixed in 3.3.1

**Comment by dentoir** [comment]
My bad, I didn't catch this new release, and indeed, this fixes my problem. Great news, thank you.
